yt/data_objects/static_output.py

```diff
@@ -1390,6 +1390,7 @@ def add_gradient_fields(self, input_field):
 
         Examples
         --------
+        with a cartesian dataset ds
```

Comment:
it might be worth adding a curvilinear example after this one since you're being explicit that this example is cartesian.

Reply:
I agree ! Rather than copy pasting the existing example with a mere 3 characters change, I left a text note.